alephclient/crawldir.py

```diff
@@ -15,14 +15,26 @@
 
 log = logging.getLogger(__name__)
 
+
 class CrawlDirectory(object):
-    def __init__(self, api: AlephAPI, collection: Dict, path: Path, index: bool = True, nojunk: bool = False):
+    def __init__(
+        self,
+        api: AlephAPI,
+        collection: Dict,
+        path: Path,
+        index: bool = True,
+        nojunk: bool = False,
+    ):
         self.api = api
         self.index = index
-        self.exclude = {
-            "f": re.compile(r"\..*|thumbs\.db|desktop\.ini", re.I),
-            "d": re.compile(r"\..*|\$recycle\.bin|system volume information", re.I)
-        } if nojunk else None
+        self.exclude = (
+            {
+                "f": re.compile(r"\..*|thumbs\.db|desktop\.ini", re.I),
+                "d": re.compile(r"\..*|\$recycle\.bin|system volume information", re.I),
+            }
+            if nojunk
+            else None
+        )
```

Comment:
Since this exclusion logic can get arbitrarily complex, I wonder if it may make sense to separate it into its own module? In that module, these path regexes could be defined as globals, too :)